lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -122,6 +122,21 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       element.appendChild(this.renderCategoryHeader(category, groups));
     }
 
+    // Budgets
```

Comment:
I think the Budgets block should follow **after** Metrics.

I know we didn't discuss this much before.. so apologies for that. :/


a few reasons why, though:

1. The top level metrics are just that... 'top level'.. and it's a bit confusing to put something above them.
1. The perf score is determined from the metrics, and their current proximity helps make this connection. Putting budgets in between adds some confusion... ("Does budget failure affect perf score?")
1. Just considering screen real-estate, the budgets block is really tall, and pushes the metrics often below the fold.

@khempenius wdyt?

Reply:
Totally fine with me (& I think that makes more sense) - I was just following the mocks.

Reply:
groovy! lets switcheroo. :)